src/metpy/calc/thermo.py

```diff
@@ -955,6 +966,19 @@ def _parcel_profile_helper(pressure, temperature, dewpoint):
     temp_upper = moist_lapse(unique[::-1], temp_lower[-1]).to(temp_lower.units)
     temp_upper = temp_upper[::-1][indices]
 
+    # Do the virtual temperature correction for the parcel above the LCL
+    if assumptions.use_virtual_temperature:
```

Comment:
Given the discussion, from which my takeaway is "Tv is about properly calculating buoyancy, not anything to do with pseudo-adiabatic temperature change", I wonder if the right place to deal with this is in `cape_cin`?

Reply:
Yeah, I think that would make more sense!

Reply:
I agree.